opensquirrel/utils/matrix_expander.py

```diff
@@ -145,6 +146,21 @@ def visit_matrix_gate(self, gate: MatrixGate) -> NDArray[np.complex_]:
         return expanded_matrix
 
 
+X = np.array([[0, 1], [1, 0]])
+Y = np.array([[0, -1j], [1j, 0]])
+Z = np.array([[1, 0], [0, -1]])
```

Comment:
```suggestion
x_axis = np.array([[0, 1], [1, 0]])
y_axis = np.array([[0, -1j], [1j, 0]])
z_axis = np.array([[1, 0], [0, -1]])
```

Trying to avoid variable names starting with uppercase if possible, although I admit `X`, `Y`, and `Z` are much more readable. Just a suggestion.

Reply:
I would prefer to conform to the pep8 style guide as much as possible, which says that constants should be `UPPER_CASE_WITH_UNDERSCORES` ([see](https://peps.python.org/pep-0008/#constants))

Reply:
OK, beautiful!